ui/packages/consul-ui/app/templates/dc/acls/auth-methods/show.hbs

```diff
@@ -27,6 +27,9 @@
       compact
         (array
           (hash label="General info" href=(href-to "dc.acls.auth-methods.show.auth-method") selected=(is-href "dc.acls.auth-methods.show.auth-method"))
+    (if (env "CONSUL_NSPACES_ENABLED")
```

Comment:
@johncowen Is this the right conditional? The demo site had it showing before I changed the scenario to be nspaces enabled. Does the demo site always have nspaces enabled?

Reply:
Yeah this is the right env var to use. You're right though, the demo site does have everything enabled by default. We should maybe change that in future now that we have our bookmarklets and URLs to be able to link to specific scenarios.

I mentioned a while back about using potentially using `can` for this type of feature flagging I saw some convo about others specifically not doing that, so maybe we should chat about it before we 100% decide.

TL;DR The conditional here is 💯